extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/exec/MSQSelectTest.java

```diff
@@ -1790,6 +1792,9 @@ public void testSelectRowsGetTruncatedInReports() throws IOException
       result.add(new Object[]{1});
     }
 
+    Map<String, Object> queryContext = new HashMap<>(context);
```

Comment:
Can you add a testcase which denotes what would happen if we have the default parameter?

Reply:
Added a test which asserts that the expected results has more than Limits.MAX_SELECT_ROWS